src/wdio/services/TerraService.js

```diff
@@ -41,6 +42,9 @@ export default class TerraService {
       /* `describeViewports` provides a custom Mocha `describe` block for looping test viewports. */
       describeViewports: viewportHelpers.describeViewports,
 
+      /* `hideInputCaret` hides the blinking input caret that appears in editable text areas. */
```

Comment:
nit: input or text areas

Reply:
Changed in cce5091cbdebce172e5f5c54d0950f86134a8b33